astrology.lic

```diff
@@ -101,6 +102,7 @@ class Astrology
       return if ['detect any portents'].include? result
       pools = check_pools
       break if pools['future events'] == prev_size
+      break if pools['future events'] == 10
```

Comment:
It could be more useful to have a `max?` function for a Pools class. Potential refactoring :]